### ISSUE TITLE
cant join team in a game

### ISSUE THREAD

**Issue by NikolasHaze** [designer1337/csgo-cheat-base#123]
i cant join a team in a game, how do i fix this?

**Comment by pessiuff** [comment]
sendpacket inside createmove needs to be get with a different method and the problem will be fixed

**Comment by astroux** [comment]
how do I do that?

**Comment by pessiuff** [comment]
hook createmove with createmove proxy rather than normal createmove

**Comment by NikolasHaze** [comment]
> how do I do that?

i already fixed the problem, but im giving you the solution.
look at this base and use the same function it works, if you need help reply to me.
https://github.com/cazzwastaken/based/blob/master/src/core/hooks.cpp

**Comment by pessiuff** [comment]
> > how do I do that?
> 
> i already fixed the problem, but im giving you the solution. look at this base and use the same function it works, if you need help reply to me. https://github.com/cazzwastaken/based/blob/master/src/core/hooks.cpp

yeah removing the sendpacket is actually a good idea of fixing the issue but for making tickbase stuff, you'll need to get sendpacket using something

**Comment by astroux** [comment]
@NikolasHaze im still confused.

**Comment by NikolasHaze** [comment]
> @NikolasHaze im still confused.

@astroux send me your discord

**Comment by maecry** [comment]
> i already fixed the problem, but im giving you the solution. look at this base and use the same function it works, if you need help reply to me. https://github.com/cazzwastaken/based/blob/master/src/core/hooks.cpp

as you mean the "globals::UpdateLocalPlayer();" function ?
I'm also confused as hell, can you help me thru Kiệt#0445?

also @[pessiuff](https://github.com/pessiuff) I'd tried to hook createmove_proxy as well however it will entirely broke the cmd in this SDK 

edit: I fixed it thanks to @[NikolasHaze](https://github.com/NikolasHaze) so it's the allocKeyValue hook lol

**Comment by Neuzo** [comment]
@NikolasHaze 
There is a simple solution to that problem:
go to your  Create move hook cut out:
 `csgo::local_player = reinterpret_cast<player_t*>(interfaces::entity_list->get_client_entity(interfaces::engine->get_local_player()));`
and paste it after this if statement:
`	if (!cmd || !cmd->command_number || !interfaces::engine->is_in_game() || !interfaces::engine->is_connected() || !csgo::local_player)
		return false;`
also in the if statement above you have to remove this check: `!csgo::local_player`

**Comment by lol12345-6** [comment]
> look at this base and use the same function it works, if you need help reply to

hi mate would you be able to help me ? 
discord: Relentless#7478

**Comment by maecry** [comment]
> > look at this base and use the same function it works, if you need help reply to
> 
> hi mate would you be able to help me ? discord: Relentless#7478

https://github.com/rollraw/qo0-base/blob/92a3b3aefa220da941c3feee387702c772d1af82/base/core/hooks.cpp#L55
https://github.com/rollraw/qo0-base/blob/92a3b3aefa220da941c3feee387702c772d1af82/base/core/hooks.cpp#L197

just paste this hook lol

**Comment by roventy1337** [comment]
> 

bro can u add my discord i need help roventyrekt#2368

**Comment by roventy1337** [comment]
> > > look at this base and use the same function it works, if you need help reply to
> > 
> > 
> > hi mate would you be able to help me ? discord: Relentless#7478
> 
> https://github.com/rollraw/qo0-base/blob/92a3b3aefa220da941c3feee387702c772d1af82/base/core/hooks.cpp#L55 https://github.com/rollraw/qo0-base/blob/92a3b3aefa220da941c3feee387702c772d1af82/base/core/hooks.cpp#L197
> 
> just paste this hook lol

need help bro can u add me on discord
roventyrekt#2368

**Comment by maecry** [comment]
added and how can you fail to paste something lol
